app/helpers/pages_helper.rb

```diff
@@ -33,6 +33,15 @@ def plugin_title(plugin)
     "#{plugin.name}#{detail}"
   end
 
+  def plugin_icon(plugin)
+    registered = {
+      action: 'hand-rock-o',
```

Comment:
Please could you add an explanation about what this method is for (inline rdoc).